pykern/pkconst.py

```diff
@@ -31,3 +33,33 @@
 
 #: Copied from numconv, which copied from RFC1924
 BASE62_CHARS = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+TWO_PI = 2 * math.pi
+RT_TWO_PI = math.sqrt(2*math.pi)
+RT_2_OVER_PI = math.sqrt(2/math.pi)
+pi = math.pi
+RT_2 = math.sqrt(2.)
+TWO_PI = 2 * math.pi
+RT_TWO_PI = math.sqrt(2*math.pi)
```

Comment:
Be consistent with spacing. below you have eg `c_SQ / e` (with spaces). Here there are no spaces. The software group normally spaces things like other Python code: `math.sqrt(2 * math.pi)`. In this case, you really want, `math.sqrt(TWO_PI)`, because it's already defined.